CMakeLists.txt

```diff
@@ -106,6 +106,10 @@ IF (${VERBOSE} MATCHES "true")
     SET(CMAKE_VERBOSE_MAKEFILE ON)
 ENDIF ()
 
+IF (${TOOLS_3})
```

Comment:
use TDENGINE_3 or FOR_TDENGINE_3 is better since taosTools has only one branch now